### PR TITLE
Reduce use of 'slave' terminology in documentation

### PR DESCRIPTION
**Pull request by adinhtdsibm**
Jenkins was designed with a "master <-> slave" architecture. 'slave' is deprecated and 'agent' should be used instead.
#words-matter
https://www.ibm.com/blogs/think/2020/08/words-matter-driving-thoughtful-change-toward-inclusive-language-in-technology/

@pjdarton

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/Messages.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
-vSphereCloudSlave.OfflineReason.ShuttingDown=Shutting down VSphere Cloud Slave
+vSphereCloudSlave.OfflineReason.ShuttingDown=Shutting down VSphere Cloud Agent
```
Comment (pjdarton) [inline src/main/resources/org/jenkinsci/plugins/Messages.properties:1]: Please remove trailing whitespace.

File: src/main/resources/org/jenkinsci/plugins/vsphereCloudProvisionedSlave/configure-entries.jelly [hunk 2]
```diff
@@ -51,7 +51,7 @@
     </f:entry>
 
     <f:entry title="${%Delay between launch and boot complete}" field="launchDelay"
-    description="${%How many seconds between the VM being brought back to life before Jenkins can connect and bring it online as a slave.}" >
+    description="${%How many seconds between the VM being brought back to life before Jenkins can connect and bring it online as a agent.}" >
```
Comment (pjdarton) [inline src/main/resources/org/jenkinsci/plugins/vsphereCloudProvisionedSlave/configure-entries.jelly:54]: "a agent" -> "an agent"

File: src/test/java/org/jenkinsci/plugins/vsphere/tools/CloudProvisioningStateTest.java [hunk 3]
```diff
@@ -233,7 +233,7 @@ public void pruneUnwantedRecordsGivenUnknownTemplatesThenRemovesRecordsForEmptyD
         final String deletedButActiveNodeName = createNodeName();
         final String livedAndDiedNodeName = createNodeName();
         final CloudProvisioningState instance = createInstance();
-        // A template which the user deleted but still has an active slave
+        // A template which the user deleted but still has an active agent
```
Comment (pjdarton) [inline src/test/java/org/jenkinsci/plugins/vsphere/tools/CloudProvisioningStateTest.java:236]: Please remove trailing whitespace

**Review by pjdarton — CHANGES_REQUESTED** [review]
Minor typos:
"a agent" should be "an agent"
...and you've added a trailing space to the end of a couple of lines.  Hint: Tell your IDE/text-editor to show you EOLs and/or whitespace and this will be easier to spot.

File: src/main/resources/org/jenkinsci/plugins/Messages.properties (hunk 1 above)
Comment (adinhtdsibm) [reply src/main/resources/org/jenkinsci/plugins/Messages.properties:1]: Removed trailing space in the src/main/resources/org/jenkinsci/plugins/Messages.properties file.

File: src/main/resources/org/jenkinsci/plugins/vsphereCloudProvisionedSlave/configure-entries.jelly (hunk 2 above)
Comment (adinhtdsibm) [reply src/main/resources/org/jenkinsci/plugins/vsphereCloudProvisionedSlave/configure-entries.jelly:54]: Completed the change by modifying "a agent" -> "an agent"

File: src/test/java/org/jenkinsci/plugins/vsphere/tools/CloudProvisioningStateTest.java (hunk 3 above)
Comment (adinhtdsibm) [reply src/test/java/org/jenkinsci/plugins/vsphere/tools/CloudProvisioningStateTest.java]: Remove trailing whitespace from the src/test/java/org/jenkinsci/plugins/vsphere/tools/CloudProvisioningStateTest.java file.

**Review by pjdarton — APPROVED** [review]

**Pull request merged**